tests/usm/usm_allocate_free.h

```diff
@@ -418,17 +418,7 @@ class check_usm_allocate_free {
  */
 template <typename T, typename op>
 static void run_usm_test(util::logger &log) {
-  try {
-    check_usm_allocate_free<T, op>{}(log);
-  } catch (const sycl::exception &e) {
-    log_exception(log, e);
-    std::string errorMsg =
-        "a SYCL exception was caught: " + std::string(e.what());
-    FAIL(log, errorMsg);
-  } catch (const std::exception &e) {
-    std::string errorMsg = "an exception was caught: " + std::string(e.what());
-    FAIL(log, errorMsg);
-  }
+  { check_usm_allocate_free<T, op>{}(log); }
```

Comment:
```suggestion
  check_usm_allocate_free<T, op>{}(log);
```
We modify this line anyway, so we can drop extra braces here.